src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5360,6 +5360,12 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_NoBestNullabilityConditionalExpression_Title" xml:space="preserve">
     <value>No best nullability for operands of conditional expression.</value>
   </data>
+  <data name="WRN_MissingNonNullTypesContextForAnnotation" xml:space="preserve">
+    <value>The annotation for nullable reference types can only be used in code with a `[NonNullTypes(true)]` context.</value>
```

Comment:
`within a …`?

Reply:
Consider using single quotes around the attribute rather than backticks, for consistency with other instances of code in error messages.